src/data/leo-images.js

```diff
@@ -12,13 +12,13 @@ export const leoImages = [
   },
   {
     label: 'Bioconductor (R 3.6.1, Bioconductor 3.9, Tidyverse 1.2.1)',
-    version: '0.0.2',
-    updated: '2019-09-06',
+    version: '0.0.3',
+    updated: '2019-10-25',
```

Comment:
Something I just noticed and Pete and I dug into understanding... when you don't specify a time, UTC is assumed. If you'd rather be more specific, you can add a time using ISO standard date/time format, e.g.: `2019-10-25T16:30:00-04:00` or `2019-10-25T20:30:00Z` (which represent the same instant). Be sure to include the `T` though because this ultimately gets passed to `new Date()` and Safari's implementation rejects it without the `T`.

Further info: Chrome, Firefox, and Safari assume UTC if you don't specify a time. Safari also assumes UTC if you specify a time with no time zone. Chrome and Firefox, however, when given a specific time assume _local time zone_!!! Weird, huh?

TL;DR: You don't have to specify a time if you're okay with midnight UTC. If you do specify a time, use correct ISO format with time zone.

Reply:
I don't think this timestamp is just informational..So I'll leave it as is